src/Core/FlatpakBackend.vala

```diff
@@ -72,6 +74,12 @@ public class AppCenterCore.FlatpakBackend : Backend, Object {
         appstream_pool = new AppStream.Pool ();
         appstream_pool.set_cache_flags (AppStream.CacheFlags.NONE);
         package_list = new Gee.HashMap<string, Package> (null, null);
+        monitor = installation.create_monitor ();
```

Comment:
Check if `installation` is `null` before using it and wrap this whole block in a `try/catch` as `create_monitor` can throw errors.